docs/product-user-guide/src/main/asciidoc/business-central-settings-proc.adoc

```diff
@@ -0,0 +1,7 @@
+[[_business_central_settings_proc]]
+= Business Central Settings
+
+You can customize several {PRODUCT} settings.
+
+.Procedure
+Click the gear icon in the upper right of the screen to display the Settings perspective.
```

Comment:
Since it is a taskbar, shouldnt it be "Click the gear icon **on** the upper right of the screen..."?

Reply:
As before